src/types/types.js

```diff
@@ -858,6 +858,8 @@ export type EdgeSwapApproveOptions = {
  * If a provider can satisfy a request, what is their price?
  */
 export type EdgeSwapQuote = {
+  +request: EdgeSwapRequest,
```

Comment:
Shouldn't this be optional so that we don't break all the plugins that return an EdgeSwapQuote but without the `request` object

Reply:
Right, but notice that we jam the request into the object on the way out to the GUI. So, if we don't update the plugins, the GUI still gets the object. This will be a breaking change for the swap plugin from a types perspective, but it will be an easy fix.